### PR TITLE
Added tests

### PR DESCRIPTION
**Pull request by r-prykhodko**

### CODE REVIEW

**Review by djtyrf312 — CHANGES_REQUESTED** [review]
remove json files from your pr and use boundary values technique for these tests

File: src/getCoinCombination.test.js [hunk 1]
```diff
@@ -1,10 +1,59 @@
...
+
+  it('should return [1, 0, 0, 0] for 1 cent', () => {
+    expect(getCoinCombination(1)).toEqual([1, 0, 0, 0]);
+  });
+
+  it('should return [0, 0, 0, 1] for 25 cents', () => {
+    expect(getCoinCombination(25)).toEqual([0, 0, 0, 1]);
+  });
+
+  it(`should return [0, 0, 0, 1] for 26 cents
```
Comment (djtyrf312) [inline src/getCoinCombination.test.js:22]:
```suggestion
  it(`should return [1, 0, 0, 1] for 26 cents
```

File: src/getCoinCombination.test.js [hunk 2]
```diff
@@ -1,10 +1,59 @@
...
+    (handling edge case just beyond 25 cents)`, () => {
+    expect(getCoinCombination(26)).toEqual([1, 0, 0, 1]);
+  });
+
+  it(`should return [1, 0, 0, 1] for 31 cents
```
Comment (djtyrf312) [inline src/getCoinCombination.test.js:27]:
```suggestion
  it(`should return [1, 1, 0, 1] for 31 cents
```

**Review by djtyrf312 — CHANGES_REQUESTED** [review]
you have just removed json files from your project if this pr merges the project would be broken. You need to reset commit with changes in that files and remove them from index, then make a new commit only with test file. Also, you need to make some fixes in the tests names
